docs/book/features/raw_block_volume.md

```diff
@@ -1,7 +1,7 @@
 # vSphere CSI Driver - Single-Access, Block Based Volume (Raw Block Volume)
 
 [Raw Block Volume](https://kubernetes.io/docs/concepts/storage/persistent-volumes/#raw-block-volume-support) feature in Kubernetes was promoted to stable in Kubernetes 1.18.
-vSphere CSI Driver release `v2.3.0` onwards has Raw Block Volume feature released as `Alpha`. We do not recommend `Alpha` features for production use. Raw Block Volume feature is only supported for linux based nodes.
+vSphere CSI Driver release `v3.0` onwards has Raw Block Volume feature released as `GA`. Raw Block Volume feature is only supported for linux based nodes.
```

Comment:
Remove this MD file, as we will publish doc at VMware Official CSI Driver Doc.

Reply:
We can remove this file after Release of 3.0 and Documentation.

Reply:
done